doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -333,27 +341,27 @@ explaining the cause.
 
 The following strings must not be used as attribute names:
 
-* `id`, as it would conflict to the field used to represent entity id)
+* `id`, as it would conflict to the field used to represent entity id.
 
-* `type`, as it would conflict to the field used to represent entity type)
+* `type`, as it would conflict to the field used to represent entity type.
 
 * `geo:distance`, as it would conflict with the string used in `orderBy` for proximity to center point.
 
-* Virtual attribute names (see specific section about virtual attributes)
+* Virtual attribute names (see specific section on "Virtual Attributes")
 
-## Ordering results
+## Ordering Results
 
 Operations that retrieve lists of entities allow the `orderBy` URI parameter to specify the order; its 
```

Comment:
to specify which attributes or properties will be used as criteria to order results